src/core/EntryAttachments.cpp

```diff
@@ -34,9 +35,9 @@ bool EntryAttachments::hasKey(const QString& key) const
     return m_attachments.contains(key);
 }
 
-QList<QByteArray> EntryAttachments::values() const
+QSet<QByteArray> EntryAttachments::values() const
```

Comment:
I was wondering why this change is necessary, but then I saw that the only two usages of this method also convert the list to a set, so I'm ok with this change.